Procfile

```diff
@@ -1 +1 @@
-web: urinception-web
+web: go run main.go
```

Comment:
Will this work? I assumed the go buildpack would not put the go tooling into the final slug

Reply:
Hmm that sounds right.

Reply:
https://github.com/heroku/urinception/pull/11/commits/2174bbb723b750e37bb1cc16b467f5bba77aa4d6